common/app/assets/assets.scala

```diff
@@ -5,7 +5,7 @@ import org.apache.commons.io.IOUtils
 import play.api.{ Mode, Play }
 import play.api.libs.json.{ JsString, Json, JsObject }
 import conf.Configuration
-import collection.mutable.{ Map => MutableMap }
```

Comment:
I'm glad you've used `ConcurrentMap` here ... I'm guessing this other `MutableMap` should be a `ConcurrentMap`, too, though. Although that could be sorted in a separate refactor pull request.